modules/cron/main.tf

```diff
@@ -195,8 +195,9 @@ resource "google_cloud_run_v2_job_iam_binding" "authorize-calls" {
 }
 
 // project iam, as job iam does allow user to actually list the job to access it
+// only grant it to groups, individual should have access otherwise.
 resource "google_project_iam_member" "authorize-list" {
-  for_each = toset(var.invokers)
+  for_each = toset([for i in var.invokers : i if startswith(i, "group:")])
```

Comment:
rather than silently ignoring non-groups, use a lifecycle assertion?

Reply:
the reason i went with silently ignoring non-groups is because, im using `var.invokers` for granting `run.invoker` and only grant `run.viewer` at the project level for groups.